src/Tribe/Ticket_Object.php

```diff
@@ -684,6 +716,14 @@ public function capacity() {
 				return '';
 			}
 
+			/* @var Tribe__Cache $cache */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: ab21a1575853d7e6410a8f33f52ee436</sub>